services/src/main/java/org/keycloak/authorization/admin/ResourceServerService.java

```diff
@@ -90,9 +94,11 @@ public ResourceServer create(boolean newClient) {
         return resourceServer;
     }
 
+    @Path("/update")
```

Comment:
It seems to me that this line was added by mistake. Once I remove it, the tests start to pass.

```suggestion
```

Reply:
Oh, good catch! Yeah I think this fixes the test suite, thanks :)